langchain4j-cloud-sql-postgres/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.36.0-SNAPSHOT</version>
+        <relativePath>../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+    <properties>
+        <pgvector-java.version>0.1.4</pgvector-java.version>
+        <postgresql.version>42.7.0</postgresql.version>
+    </properties>
+
+    <artifactId>langchain4j-cloud-sql-postgres</artifactId>
```

Comment:
Let's simplify to langchain4j-cloud-sql-pg

Reply:
done